packages/polyfills/migration-guide.md

```diff
@@ -0,0 +1,18 @@
+# Migration guide
+
+This is a concise summary of changes and recommendations around updating `@shopify/polyfills` in consuming projects. For a more detailed list of changes, see [the changelog](./CHANGELOG.md).
+
+## [Unreleased]
+
+The following polyfills have been removed because they are provided by `core-js@3`. Please remove imports containing:
```

Comment:
```suggestion
Base polyfills now use `core-js@3`. The following polyfills have been removed because they are provided by `core-js@3`. Please remove imports containing:
```